component/attr_cache/attr_cache.go

```diff
@@ -802,7 +721,11 @@ func (ac *AttrCache) CopyFromFile(options internal.CopyFromFileOptions) error {
 		}
 		// TODO: Could we just update the size and mod time of the file here? Or can other attributes change here?
 		// TODO: we're RLocking the cache but we need to also lock this attr item because another thread could be reading this attr item
-		ac.invalidatePath(options.Name)
+
+		toBeUpdated, getErr := ac.cacheMap.get(options.Name) //empty for TestCopyFromFileDoesNotExist()
```

Comment:
```suggestion
		toBeUpdated, getErr := ac.cacheMap.get(options.Name)
```
Drop comment about test unless it's needed and clearly worded